lti/lti-common/src/java/org/sakaiproject/lti13/LineItemUtil.java

```diff
@@ -55,6 +55,7 @@
 import org.sakaiproject.lti13.util.SakaiLineItem;
 
 import org.tsugi.lti.LTIUtil;
+import org.tsugi.lti13.LTI13Util;
```

Comment:
Line 125 is now referring to a LTIUtil.toLongNull instead of the previous reference to LTI13Util.getLongNull. Therefore, this import is now superfluous. That said, I'll proceed to test the new use of LTIUtil.toLongNull.